add-on/chrome/test/mochitest/browser_mozLoop_sharingListeners.js

```diff
@@ -157,13 +157,22 @@ add_task(function* test_infoBar() {
 
   let testBarProps = function() {
     let bar = getInfoBar();
-
     // Start with some basic assertions for the bar.
     Assert.ok(bar, "The notification bar should be visible");
     Assert.strictEqual(bar.hidden, false, "Again, the notification bar should be visible");
 
+    // Message label
+    Assert.equal(bar.label, getLoopString("infobar_screenshare_no_guest_message"), "The bar label should match");
+
+    // Pause button and type
     let button = bar.querySelector(".notification-button");
     Assert.ok(button, "There should be a button present");
+    Assert.equal(button.type, "pause", "The bar button should be type pause");
+
+    // Paused state and label message
+    button.click();
+    Assert.equal(bar.label, getLoopString("infobar_screenshare_stop_no_guest_message"), "The bar label should match when paused");
+    button.click();
   };
```

Comment:
I think the way to do this would be to use the `LoopRooms._setRoomsCache()` function (see `browser_LoopRooms_channel.js` for an example).

You could set a room list via that function. To actually trigger an infobar update, you could add an extra optional parameter which calls the checkParticipantsUpdate function or something like that.